pymc/tests/test_distributions_moments.py

```diff
@@ -341,3 +342,21 @@ def test_weibull_moment(alpha, beta, size, expected):
     with Model() as model:
         Weibull("x", alpha=alpha, beta=beta, size=size)
     assert_moment_is_expected(model, expected)
+
+@pytest.mark.parametrize(
+    "c, size, expected",
+    [
+        (1, None, 1),
+        (1, 5, np.full(5, 1)),
+        (np.arange(1, 6), None, np.arange(1, 6)),
+        (
+            np.arange(1, 6),
+            (2, 5),
+            np.full((2, 5), np.arange(1,6)),
+        ),
+    ],
+)
+def test_constant_moment(c, size, expected):
```

Comment:
3 of the tests should suffice for such simple distribution. Otherwise looks good :)